library/vault

```diff
@@ -6,4 +6,5 @@
 0.7.2: git://github.com/hashicorp/docker-vault@02af9417ae42bf6290b4b36c2188b1836f1017a8 0.7.2
 0.7.3: git://github.com/hashicorp/docker-vault@02af9417ae42bf6290b4b36c2188b1836f1017a8 0.7.3
 0.8.0-beta1: git://github.com/hashicorp/docker-vault@20c626ab1c89a17c24f370ae18cfe6cbc0eb8d33 0.8.0-beta1
```

Comment:
It'll still show up in the "Supported tags" list on the Docker Hub and get rebuilt if the base image gets updated.